android/cli/commands/_build.js

```diff
@@ -3697,6 +3697,17 @@ AndroidBuilder.prototype.generateAndroidManifest = function generateAndroidManif
 		});
 	}
 
+	if (this.realTargetSDK >= 24) {
+		Object.keys(finalAndroidManifest.application.activity).forEach(function (name) {
+			var activity = finalAndroidManifest.application.activity[name];
+			if (!activity.configChanges) {
+				activity.configChanges = ['density'];
```

Comment:
No, there other values that go into `configChanges` but it is a requirement for consistency across different Android 7 devices for zooming in/out. For `activity.configChanges = ['density'];` I used the same logic used elsewhere just in case a situation arises where `configChanges` is empty but the fact is `screenSize` will always be added to `configChanges` so it is kind of redundant in a way.

Also, I should make users aware that `density` is added by default?